source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -236,7 +236,21 @@ struct apply_impacts_visitor : public boost::static_visitor<> {
             this->operator()(line);
         }
     }
-    void operator()(const nt::StopArea* ) {
+
+
+    template<typename T>
+    static std::string make_uri(const std::string& ref_uri, T uris) {
+        auto adapted_uri = ref_uri + ":adapted-";
+        for (size_t i=0; i < uris.size(); ++i) {
+            auto uri = adapted_uri + "-" + boost::lexical_cast<std::string>(i);
+            if (uris.find(uri) == uris.end()) {
+                return uri;
+            }
+        }
+        return ref_uri;
+    }
+
+    virtual void operator()(const nt::StopArea* ) {
         LOG4CPLUS_INFO(log4cplus::Logger::getInstance("log"),
                        "apply_impact_visitor on StopArea not implemented yet!");
     }
```

Comment:
Better to just remove this method if it is useless, it will fail at compilation time.